odxtools/service.py

```diff
@@ -55,6 +58,7 @@ def __init__(
         self.long_name: Optional[str] = long_name
         self.description: Optional[str] = description
         self.semantic: Optional[str] = semantic
+        self.admin_data: Optional[AdminData] = admin_data
```

Comment:
Existing bug unrelated to PR?

Reply:
yes, there might be a few more such minor fixes lurking. I can make these separate PRs or commits if you want (and we can find them)...

Reply:
That would be better, since it would make for a cleaner git history, and easier to make changelog